### ISSUE TITLE
Sampling rate support for statsd counters (and possibly other types)

### ISSUE THREAD

**Issue by vanatpm** [postmates/cernan#199]
Submission of statsd counters with sampling rates causes a "BAD PACKET" error. Stripping the optional sampling rate portion of the signal will allow the submission to go through as expected.

**I tried this:**

* Installing a local instance of Cernan per the [QuickStart](https://github.com/postmates/cernan/wiki/Quickstart) page.
* `echo "foo.bar:1|c|@0.1" | nc -c -u localhost 8125`

**I expected to see this happen:**

Signal acceptance

**Instead, this happened:**
```
[cernan::source::statsd][74][2017-01-25T22:28:39.120864+00:00][ERROR] BAD PACKET: "foo.bar:1|c|@0.1"
```

`echo "foo.bar:1|c" | nc -c -u localhost 8125` works fine.

**Comment by blt** [comment]
Oh fun! So that spec seems to be non-normative based on other clients'
behaviour. There's a PR open now to support this, that said, and we'll get
it merged up promptly.

Thank you for the well structured issue!

**Comment by blt** [comment]
Resolved by #198
